examples/basic/main.tf

```diff
@@ -11,14 +11,15 @@ module "resource_group" {
 }
 
 ##############################################################################
-# COS instance
+# Events-notification-instance
 ##############################################################################
 
-resource "ibm_resource_instance" "cos_instance" {
-  name              = "${var.prefix}-cos"
-  resource_group_id = module.resource_group.resource_group_id
-  service           = "cloud-object-storage"
-  plan              = "standard"
-  location          = "global"
-  tags              = var.resource_tags
+module "event_notification" {
+  source                   = "../../"
+  resource_group_id        = module.resource_group.resource_group_id
+  name                     = "${var.prefix}-en"
+  tags                     = var.resource_tags
+  plan                     = "standard"
```

Comment:
Is there a limit of lite plans? Perhaps use lite plan if there is no limit for basic example